spec/rubocop/cop/rails/read_write_attribute_spec.rb

```diff
@@ -5,7 +5,7 @@
     it 'registers an offense and corrects a symbol' do
       expect_offense(<<~RUBY)
         res = read_attribute(:test)
-              ^^^^^^^^^^^^^^ Prefer `self[:attr]` over `read_attribute(:attr)`.
+              ^^^^^^^^^^^^^^ Prefer `self[:test]` over `read_attribute(:test)`.
```

Comment:
> I wonder if we could omit the `over ... ` part because it essentially duplicates the highlighted code ❓ 

It is okay to omit the duplicated message or not. Apart from that, Can you tweak the highlight range?

```diff
     res = read_attribute(:test)
-          ^^^^^^^^^^^^^^ Prefer `self[:test]` over `read_attribute(:test)`.
+          ^^^^^^^^^^^^^^^^^^^^^ Prefer `self[:test]` over `read_attribute(:test)`.
```

Reply:
Makes sense to update the range! Done ✅ 
I've also got rid of the `over` part because it basically duplicates the highlighted piece and feels like it's unnecessary